tools/yac_clipper/yac.xml

```diff
@@ -41,7 +41,7 @@
     <outputs>
         <data format="fasta" metadata_source="input" name="output" label="Clipping of ${input.name}">
           <change_format>
-              <when input="out_format" value="fastq" format="fastq" />
+              <when input="out_format" value="fastq" format="fastqsanger" />
```

Comment:
Given that yac.py does not manipulate the quality score, I think `format_source="input"` would be more correct, while having the intended effect (that fastqsanger input produces fastqsanger output, right?)